### PR TITLE
Python package learning_rates - replacing 'eta' with 'learning_rate'

### PR DESCRIPTION
**Pull request by ceshine**
I see a similar issue has already been raised in #1054. Setting learning_rates didn't yield adaptive learning rates as expected.

In my case, changing the parameter name seems to fix the problem. (Using the latest code on master)

### CODE REVIEW

**Comment by Far0n** [comment]
That's indeed a second issue besides the reset of the seed. 
`bst.set_param({'eta': learning_rates[i]})` is not working, if you passed 'learning_rate' as parameter. On the other hand. If you pass 'eta' via the parameter dict, `bst.set_param({'learning_rate': learning_rates[i]})` works.

**Comment by ceshine** [comment]
Thanks @Far0n for your quick reply. I see now it's because I am accessing learning_rates from XGBClassifier (like in #1018 ) so 'learning_rate' is always set (being default to 0.1).

Apparently my quick hack breaks the other scenario in which 'eta' is set, so please fell free to close this PR and go with your solution. :)

**Comment by Far0n** [comment]
@ceshine I just created a new PR. Thx for pointing this issue.
